doc/user/content/sql/create-source/csv-s3.md

```diff
@@ -8,8 +8,10 @@ menu:
 {{< experimental >}}The S3 source type{{< /experimental >}}
 
 {{% create-source/intro %}}
-This document details how to connect Materialize to an S3 Bucket that contains text- or
-byte-encoded CSV-serialized objects.
+This document details how to connect Materialize to an S3 Bucket that contains
+multiple objects, and to listen for new object creation. Each S3 object can
+contain multiple records serialized as CSV. It will treat newline-delimited rows
```

Comment:
The "it" in the last sentence is unclear. Any reason not to use the phrasing used for the other source types: "Each S3 object can contain multiple records serialized as CSV, separated by newlines."

Reply:
done, yeah not sure why I changed it here.